modules/pytket-qiskit/pytket/extensions/qiskit/backends/aer.py

```diff
@@ -660,7 +660,7 @@ def _process_model(noise_model: NoiseModel, gate_set: Set[OpType]) -> dict:
             link_errors_qubits.add(q0)
             link_errors_qubits.add(q1)
             # to simulate a worse reverse direction square the fidelity
-            link_errors[(q1, q0)].update({optype: 1 - gate_fid ** 2})
+            link_errors[(q1, q0)].update({optype: 1 - gate_fid**2})
```

Comment:
has the default black formatting changed?

Reply:
Yes, this is a change in the latest version of black.